### ISSUE TITLE
Add possible filters in metadata objects

### ISSUE THREAD

**Issue by MansMeg** [statisticssweden/PxWeb#33]
Hi!

I have gotten requests for the pxweb R package to enable the use of different aggregations. This does not seem to come as a part of the pxweb metadata object and hence I do not know what possible filters and aggregations that exists for the specific variable. Would it be possible to supply that through the API as well - ie what filter variables that exists?

Example:
http://api.scb.se/OV0104/v1/doris/sv/ssd/START/BE/BE0001/BE0001D/BE0001T05AR
has the filter "vs:Flickor10", but this cannot be seen from the metadata object.

**Comment by likp** [comment]
Hi and sorry for the late reply,

Yes this is a known limitation in the current API many other has hade similar request as you. But we have decided to put most our effort and improvements int the version 2 of the API (please see the specs on https://github.com/statisticssweden/PxApiSpecs/blob/master/specs.md). Therefore we will not make any changes into the existing API.

**Comment by MansMeg** [comment]
Hi!

That sounds reasonable. Since pxweb through R has a lot of users (rougly 600 downloads per month) it would be important for me to transfer them seemlessly over to v2 of the API. Would you have the possibility to help out with that? I guess I can read the docs, but it would also be good to tet a feeling of how you plan the transition. Would that work for you? Im currently at Uppsala University, so you can find my e-mail there.

**Comment by likp** [comment]
We will try to be more transparent with our plans right here on Github as soon as possible. But in short we have once again postponed the development of the API. As it looks right now we will not start the implementation of it this year. But hopefully in the beginning of 2021. I can only answer for how Statistics Sweden is transitioning into the 2.0 version of the API an that is a rather smooth transition where we will run both version in parallel for a good set of time (probably a year of so).

**Comment by MansMeg** [comment]
Thats great! Let me know when you have more info.

**Comment by likp** [comment]
Yes I do.

**Comment by likp** [comment]
This will be handled in PxWeb API 2.0
